src/assignment/views/AssignmentResponseEdit/AssignmentResponseEdit.scss

```
@@ -80,34 +80,38 @@
 			max-height: 390px;
 		}
 	}
+}
 
-	.c-assignment-heading {
-		@media screen and (min-width: $g-bp2) {
-			position: sticky;
-			top: $g-bar-size-regular;
-			z-index: $g-c-local-toolbar-z;
-
-			&--scrolled {
-				.c-assignment-heading__top {
-					.c-toolbar__left {
-						margin-bottom: 0;
+body:not(.c-interactive-tour--in-progress) {
+	.c-assignment-response-page {
+		.c-assignment-heading {
```

OPTIONAL: if you don't add any styles on each level, you can combine the selectors
```suggestion
body:not(.c-interactive-tour--in-progress) .c-assignment-response-page .c-assignment-heading {
```